test/integration/modules/edgecore/eventbus/eventbus_suite_test.go

```diff
@@ -0,0 +1,124 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package intergration
+
+import (
+	"crypto/tls"
+	"io"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/kubeedge/kubeedge/test/integration/utils/common"
+	"github.com/kubeedge/kubeedge/test/integration/utils/edge"
+
+	MQTT "github.com/eclipse/paho.mqtt.golang"
```

Comment:
Does https://github.com/256dpi/gomqtt meet your need? We have already vendored this mqtt library.

Reply:
@m1093782566 https://github.com/256dpi/gomqtt is a mqtt broker library whereas github.com/eclipse/paho.mqtt.golang is a mqtt client library.

Reply:
I think @qizha told me that `256dpi/gomqtt` consist of both client and broker. Can we double check it?

Reply:
@m1093782566 , since event_bus is already imported and using github.com/eclipse/paho.mqtt.golang for client connection, we feel integration suite also can use the same package for client connection.

Reply:
Okay, I buy it.